client/package-lock.json

```diff
@@ -6358,6 +6358,11 @@
         "string-width": "2.1.1"
       }
     },
+    "tail": {
+      "version": "1.2.3",
+      "resolved": "https://registry.npmjs.org/tail/-/tail-1.2.3.tgz",
+      "integrity": "sha1-sI1vp5+5KIaWMaNBpRwUSXwcQlU="
+    },
```

Comment:
are you using a version of NPM (5+) that takes care of that file in production ?

it's just to be sure, to avoid security issues due to dangerous versions